tools/pot/openvino/tools/pot/algorithms/quantization/accuracy_aware_gna/algorithm.py

```diff
@@ -24,10 +26,13 @@ def __init__(self, config, engine):
         super().__init__(config, engine)
         self._config['annotation_free'] = False
         self._config['convert_to_mixed_preset'] = False
+        self._config['unify_conv_activation_weight'] = self._config.get('unify_conv_activation_weight', True)
```

Comment:
A couple of comments:
- How this unification is supposed to be controlled? by the user? I don't think this is a good idea. I would say we just need to introduce an additional HW target. Like 3.0 and 3.5 can share the same config but there will be and additional logic in the code that can enable the unification. Another option is to go with the existing workflow and introduce a separate HW configs for 3.0 and 3.5.
- Can we come up with a more self-explainable name? E.g., `unify_conv_precisions`.

Reply:
Support 👍 
Control of unification via HW config based on the device is the better way there.

Reply:
Okay, I'll add GNA3.5 to make less confuse the user.

Reply:
I changed 'unify_conv_precisions' to 'unify_qbit_of_convolution_inputs'.
If there is any better suggestion please let me know.